bokehjs/test/models/plots/plot.coffee

```diff
@@ -74,6 +74,9 @@ describe "Plot", ->
       @p._height.setValue(100)
       expect(plot_view.get_width()).to.be.equal 20
 
+    ### XXX: If you write tests like this, expect this to change every time implementation changes.
+    #        This needs to be updated, but that means rewriting this to use an actual DOM node.
```

Comment:
I don't wan't to waste time right now fixing this. This will have to be done eventually anyway when rewriting this in TypeScript.